### PR TITLE
feat: configure page paths via options

### PR DESCRIPTION
**Pull request by notrab**
Adds config to change the path to the catalog pages.

### CODE REVIEW

File: packages/gatsby-theme-moltin/gatsby-config.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 module.exports = themeOptions => {
-  const { client_id } = themeOptions
+  const { client_id, clientId } = themeOptions
 
   return {
     plugins: [
       {
         resolve: `@moltin/gatsby-source-moltin`,
-        options: { client_id }
+        options: { client_id: client_id || clientId },
```
Comment (notrab) [inline packages/gatsby-theme-moltin/gatsby-config.js:10]: So, we should support both and phase out `client_id`.

Comment (ynnoj) [reply packages/gatsby-theme-moltin/gatsby-config.js:10]: @notrab This should be reflected at [`gatsby-source-moltin`](https://github.com/moltin/gatsby-source-moltin) too.

**Review by ynnoj — APPROVED** [review]

File: packages/gatsby-theme-moltin/gatsby-config.js (hunk 1 above)
Comment (notrab) [reply packages/gatsby-theme-moltin/gatsby-config.js:10]: @ynnoj Yep! Done, will submit another PR later.

**Pull request merged**